### DISCUSSION TITLE
Deliverable 1: what format?

### DISCUSSION THREAD

**Discussion by fmaussion** [IACS-cryo/Delineation-WG#14] [General]
The proposal describes deliverable 1 as a *Review report outlining the current situation, extracting the relevant information and suspected double counting in previous studies and IPCC report and proposed solutions to improve the situation*. 

This was written as such for two reasons:
1. while it is widely acknowledged that there is double counting, the *scale* of the problem (i.e. the actual numbers) of past double counting are unknown. 
2. the proposal does not yet outline a clear way to deal with the issue. Indeed, we may need some time to reflect (and this report might help) to draw a path forward, including responsibilities and volunteers.

(1) is partly addressed by the table in the proposal (i.e. the problem is acknowledged). The effort of producing or tracking down the numbers however might be very challenging and time consuming. If done well, it could lead to a first "review publication".

(2) is the core of our work here and should be addressed / laid out over the first months of the WG.

I would suggest deliverable 1 to be written in the form of a report here on github, with inputs from the community. Ideally, it would entail responsibilities, such as: "X agreed to work on the mapping of Y over the course of 2024", etc.

Happy to hear your thoughts about:
- to what extent (1) is possible or necessary
- if there is enough motivation / reasons to write a review paper before the work in the group even starts (Ken and I think rather not)

**Comment by mankoff** [comment]
I just looked into the `Not just for programmers: How GitHub can accelerate collaborative and reproducible research in ecology and evolution` https://doi.org/10.1111/2041-210X.14108 paper a bit more and they wrote it using a GitHub workflow. The "Not just for programmers" paper is built from https://github.com/SORTEE-Github-Hackathon/manuscript but the key technology it uses is `manubot`.

> Manubot is a system for writing scholarly manuscripts via GitHub. Manubot automates citations and references, versions manuscripts using git, and enables collaborative writing via GitHub. An [overview manuscript](https://greenelab.github.io/meta-review/) presents the benefits of collaborative writing with Manubot and its unique features. The [rootstock repository](https://git.io/fhQH1) is a general purpose template for creating new Manubot instances. See [USAGE.md](https://github.com/greenelab/meta-review/blob/master/USAGE.md) for documentation how to write a manuscript.

Link to manubot: https://github.com/greenelab/meta-review <-- can clone and use here if we decide this is better than Overleaf or Google Docs.

**Comment by mankoff** [comment]
I've created a Milestone for D1: https://github.com/IACS-cryo/Delineation-WG/milestones

Also two other milestones, one for delineation, one for basins. I'm think Proposal item D3 "Algorithms" should be merged with data. Code and data should not be separated.

**Comment by hgoelzer** [comment]
I would think https://doi.org/10.1017/jog.2023.1 is a good enough motivation for our group and gives sufficient detail about the size of the problem, at least for Antarctica. Actual numbers for double counting would be interesting, but may be much easier to establish in hindsight once we have a first delineation worked out. 
A detailed problem statement and approach is certainly needed, but I see the forward looking part in this (what do we do, and how) much more important than the backwards looking part.

**Comment by willkochtitzky** [comment]
Thanks for starting this Fabien. I think it would be worthwhile doing the GIS work to identify the exact areas we are talking about here that are "in conflict". I am happy to help with this work, but it would be useful to make sure we are all on the same page with the datasets we are using, I am somewhat familiar with the problem in Greenland, but less so in Antarctica. Perhaps this could be a good topic of discussion for a call soon? Happy new year all!

**Reply by oien2** [reply]
I would also work on this, but I am also more familiar with Greenland than Antarctica. Looking forward to the call.
